src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -287,6 +349,690 @@ void AssimpImporterTest::openDataFailed() {
     CORRADE_COMPARE(out.str(), "Trade::AssimpImporter::openData(): loading failed: No suitable reader found for the file format of file \"$$$___magic___$$$.\".\n");
 }
 
+namespace {
+
+/* This does not indicate general assimp animation support, only used to skip
+   tests on certain versions and test files. */
+bool supportsAnimation(const std::string& fileName) {
+    /* 5.0.0 supports all of Collada, FBX, glTF */
+    if(ASSIMP_IS_VERSION_5)
+        return true;
+
+    /* splitExtension returns empty extensions for files starting with a dot */
+    const std::string extension = Utility::Directory::splitExtension("x" + fileName).second;
+    const bool isGltf = extension == ".gltf" || extension == ".glb";
+    if(isGltf) {
+        return false;
+    } else {
+        const unsigned int version = aiGetVersionMajor()*100 + aiGetVersionMinor();
+        CORRADE_ASSERT(extension == ".dae" || extension == ".fbx", "Unsupported file type", false);
```

Comment:
Could be just this, asserts with messages are mainly targeted at library users so they get a human readable description of what they did wrong. Not the case here :)

```suggestion
        CORRADE_INTERNAL_ASSERT(extension == ".dae" || extension == ".fbx");
```

Reply:
Done